### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by AzulGarza** [Nixtla/neuralforecast#328] [Announcements]
## What's Changed
* [FIX] Update license in https://github.com/Nixtla/neuralforecast/pull/285
* [FEAT] Exogenous variables in https://github.com/Nixtla/neuralforecast/pull/286
* scalers class in https://github.com/Nixtla/neuralforecast/pull/288
* General documentation improvements in https://github.com/Nixtla/neuralforecast/pull/287
* Fixed README links and added SoTA runs to examples in https://github.com/Nixtla/neuralforecast/pull/291
* Improved documentation in https://github.com/Nixtla/neuralforecast/pull/293
* Improved documentation in https://github.com/Nixtla/neuralforecast/pull/297
* Improved RNN-based/BaseRecurrent/Windows in https://github.com/Nixtla/neuralforecast/pull/298
* Save load  in https://github.com/Nixtla/neuralforecast/pull/292
* Fix normalizers  in https://github.com/Nixtla/neuralforecast/pull/301
* Improved example notebooks, changed numeration  in https://github.com/Nixtla/neuralforecast/pull/302
* Static variables in https://github.com/Nixtla/neuralforecast/pull/305
* Turned Pytorch losses into `torch.nn.module` classes  in https://github.com/Nixtla/neuralforecast/pull/311
* Changed enable_checkpointing to False default in https://github.com/Nixtla/neuralforecast/pull/309
* Correct main link pointers in https://github.com/Nixtla/neuralforecast/pull/314
* Rnns normalizers  in https://github.com/Nixtla/neuralforecast/pull/316
* rnns with decoders, autos, usage examples, fix val  in https://github.com/Nixtla/neuralforecast/pull/320
* Improved documentation3 in https://github.com/Nixtla/neuralforecast/pull/322
* getting started with LSTM and NHITS in https://github.com/Nixtla/neuralforecast/pull/323
* recovered MQ-NHITS in https://github.com/Nixtla/neuralforecast/pull/327

**Full Changelog**: https://github.com/Nixtla/neuralforecast/compare/v1.0.0...v1.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Nixtla/neuralforecast/releases/tag/v1.1.0'>v1.1.0</a>.</em>
